### PR TITLE
Issue #370 Draft of updated GSG Intro paragraph

### PR DESCRIPTION
**Pull request by inoxx03**
Proposed update to the Intro paragraph for the Getting Started Guide.

Needs review, so comments are welcome:)

If you think the suggested edits seem overly promotional in tone, please let me know and I'll tone it down:)

### CODE REVIEW

File: docs/topics/getting-started-intro.adoc [hunk 1]
```diff
@@ -1,4 +1,8 @@
...
+When starting to create cloud-native applications and services, you need an local development environment that matches the target cloud environment.
+The {OpenShiftLocal} provides an out-of-the-box solution for setting up a cloud environment on your local machine without the need for extensive setup and configuration, while the {launcher} application gives you a hassle-free way of creating functional example applications called Missions and easily building and deploying them to a cloud environment.
```
Comment (Ladicek) [inline docs/topics/getting-started-intro.adoc]: So the previous sentence puts "local development environment" and "cloud environment" to contrast, and this sentence uses "cloud environment on your local machine". I'd rather reword to something like "provides an out-of-the-box solution for setting up _such_ environment on your local machine".

File: docs/topics/getting-started-intro.adoc [hunk 2]
```diff
@@ -1,4 +1,8 @@
...
+The Missions are designed to showcase different fundamental pieces of building cloud-native applications and services, such as creating HTTP APIs, integrating you applications with a relational database to provide persistent data storage, implementing the health check and circuit-breaker patterns to ensure that your services can withstand overload and network issues, externalizing the configuration of your applications to make them more secure and easier to scale, and securing your applications with Red Hat Single Sign-on to provide authentication and authorization functionalities.
+Each mission comprises different boosters that show how to implement the same mission in different runtimes and can be updated or extended for your own use case.
```
Comment (Ladicek) [inline docs/topics/getting-started-intro.adoc]: Maybe capitalize _Booster_, like Mission was capitalized above? To show that it's a new term.

File: docs/topics/getting-started-intro.adoc (hunk 1 above)
Comment (inoxx03) [inline docs/topics/getting-started-intro.adoc]: Good point, thanks. Will fix:)

**Comment by rhoads-zach** [comment]
merged: 2a6fe94d8d5c6dff76c6980f4af42f9c737d0ecd

**Pull request closed without merging**